wire-ios/Wire-iOS/Sources/UserInterface/Availability/UIAlertController+Availability.swift

```diff
@@ -23,9 +23,24 @@ import WireDataModel
 extension UIAlertController {
 
     static func availabilityExplanation(_ availability: AvailabilityKind) -> UIAlertController {
+        var title: String = ""
+        var message: String = ""
+
+        switch availability {
+        case .none:
+            title = L10n.Localizable.Availability.Reminder.None.title
```

Comment:
If you make a typealias with` L10n.Localizable.Availability.Reminder`, you will reduce the amount of duplicate strings.

Reply:
Done!